hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -550,43 +550,6 @@ object TestUtils {
     new MatrixTable(hc, MatrixRead(reader.fullType, dropSamples, false, reader))
   }
 
-  def importBgens(hc: HailContext,
-    files: Seq[String],
-    sampleFile: Option[String] = None,
-    includeGT: Boolean = true,
```

Comment:
I like the new handling of includeGT, GP, etc; seems neater to store in (mr: MatrixRead).typ.entryType